bin/runtime-common/src/messages.rs

```diff
@@ -499,39 +328,22 @@ pub mod source {
 			let route = T::build_destination();
 			let msg = (route, msg.take().ok_or(SendError::MissingArgument)?).encode();
 
-			let fee = estimate_message_dispatch_and_delivery_fee::<T::MessageBridge>(
-				&msg,
-				T::MessageBridge::RELAYER_FEE_PERCENT,
-				None,
-			);
-			let fee = match fee {
-				Ok(fee) => fee,
-				Err(e) => {
-					log::trace!(
-						target: "runtime::bridge",
-						"Failed to comupte fee for XCM message to {:?}: {:?}",
-						T::MessageBridge::BRIDGED_CHAIN_ID,
-						e,
-					);
-					*dest = Some(d);
-					return Err(SendError::Transport(e))
-				},
-			};
-			let fee_assets = MultiAssets::from((Here, fee));
+			// let's jsut take fixed (out of thin air) fee per message in our test bridges
+			// (this code won't be used in production anyway)
```

Comment:
@bkontur Am I right here? Do you have some branch where you're using those blob dispatcher and exporter? So in the end we won't need this custom `SendXcm` implementation, right?

Reply:
(I mean #1576)

Reply:
That's super cool, thank you!